src/libraries/Native/Unix/LdapProxy.Native/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+project(LdapProxy.Native C)
```

Comment:
Nearly all shims under libraries/Native are proxies (trivial wrappers) like this one, but we do call them proxies. Would `System.DirectoryServices.Native` be a better name for this?

Reply:
I was trying to do that but it looked for me that Ldap related name shows what inside that library better than `System.DirectoryServices.Native`, but it is probably for me who did not work `System.DirectoryServices`. I will rename it.

Reply:
Ok, I can't find a solution for it, `System.DirectoryServices.Native` is Linux only so it does not fail like this on Windows. I could delete windows's proxy and use __arglist in C# for Windows.
It means I need to deal with Linux failures first, there sometimes we can't find "ldap" library, [example](https://dev.azure.com/dnceng/public/_build/results?buildId=1069352&view=logs&jobId=6f914673-aa14-50ac-ac3d-b77f2ba1f40c&j=5daea449-3d99-50b1-0b10-6781b34bf15a&t=93ec59b1-0687-557a-725b-96c98ed366c9). 
I am using `find_library(LIBLDAP NAMES ldap)` there, the name could be `libldap-2.4.so.2` on Linux, `libldap.dylib` on OSX  and it works in "default" configs but not in "SingleFile/AllSubsets_Mono" etc, am I missing something?

Reply:
I agree with @jkotas about naming this library `System.DirectoryServicies.Native`. It fits the model of the files we ship.